solutions/pytorch_tests/Dockerfile

```diff
@@ -1,10 +1,11 @@
 ARG BASE_IMAGE=python:3.9-buster
-ARG PYTORCH_VERSION=v1.10.0
 
 FROM ${BASE_IMAGE} as test
+ARG TAG=v1.10.0
+
 RUN pip3 install pytest expecttest hypothesis && \
     pip3 install torch==1.10.0+cpu torchvision==0.11.1+cpu torchaudio==0.10.0+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html
 
 WORKDIR /workspace
-RUN git clone -b ${PYTORCH_VERSION} https://github.com/pytorch/pytorch.git --single-branch --depth 1
```

Comment:
Replacing PYTORCH_VERSION here will break the ability to specify it in https://github.com/deislabs/mystikos/blob/57fca89e5e60a88a452b2bb78fba55ae1c4a73f0/solutions/pytorch_tests/Makefile#L18

You can get around this and keep your tag idea by doing something like this:
```
ARG TAG=v1.10.0
ARG PYTORCH_VERSION=${TAG}

RUN git clone -b ${PYTORCH_VERSION} ...
```
So this way you can define TAG, or PYTORCH_VERSION, or none of them and still have a successful build.